staging/src/k8s.io/kubectl/pkg/cmd/describe/describe.go

```diff
@@ -109,48 +170,15 @@ func NewCmdDescribe(parent string, f cmdutil.Factory, streams genericclioptions.
 		Example:               describeExample,
 		ValidArgsFunction:     completion.ResourceTypeAndNameCompletionFunc(f),
 		Run: func(cmd *cobra.Command, args []string) {
-			cmdutil.CheckErr(o.Complete(f, cmd, args))
-			cmdutil.CheckErr(o.Validate())
+			o, err := flags.ToOptions(parent, args)
+			cmdutil.CheckErr(err)
```

Comment:
I'd prefer to preserve `Validate` function even if it does not validate anything.

Reply:
Yeah sounds good 👍🏻

Reply:
Where is this `cmdutil.CheckErr(o.Validate())` code block?

Reply:
I have updated the patch for that, thanks!